src/invoices/PrintVoucher/usePrintData.js

```diff
@@ -12,20 +13,30 @@ const usePrintData = (invoice = {}) => {
   const { isVoucherLinesLoading, voucherLines } = useVoucherLines(voucher.id);
   const { isBatchGroupLoading, batchGroup } = useBatchGroup(voucher.batchGroupId);
   const { vendor, isLoading: isLoadingVendor } = useVendor(voucher.vendorId);
```

Comment:
it's just an example that you don't need to fetch vendor in useDefaultAccountingCode